products/product_blocks/core_link.py

```diff
@@ -30,9 +28,7 @@
 T = TypeVar("T", covariant=True)
 
 
-class ListOfPorts(SubscriptionInstanceList[T]):
-    min_items = 2
-    max_items = 2
+ListOfPorts = Annotated[list[SI], Len(min_length=2, max_length=2)]
```

Comment:
`Len(min_length=2, max_length=2)` could be written as `Len(2, 2)` as well, or `MinLen(2), MaxLen(2)`. @mrijk @hanstrompert  do you have a preference?

Reply:
I do not ike `Len(2, 2)`. The other two are fine.